lambda/custom/index.js

```diff
@@ -3,17 +3,49 @@ const Alexa = require('ask-sdk-core')
 const LaunchRequestHandler = {
   canHandle(handlerInput) {
     return handlerInput.requestEnvelope.request.type == 'LaunchRequest'
```

Comment:
Why is it == on this line rather than === ? Does it specifically need to be just double?

Reply:
O.O should be triple equal. I'll have An update before merging